.rubocop.yml

```diff
@@ -27,3 +27,4 @@ AllCops:
     - 'mqtt.gemspec'
     - 'examples/**'
     - 'spec/**/*'
+    - 'vendor/**/*'
```

Comment:
Why add the vendor directory to rubocop?

Reply:
This ignores `vendor`, and it’s necessary because on CI the gems are downloaded there, and would otherwise be checked with RuboCop

Reply:
Ah sorry! The Github diff wasn't very helpful and I thought this was in the Include section